### ISSUE TITLE
[Suggestion]hopperNoCooldown

### ISSUE THREAD

**Issue by Neubulae** [gnembon/carpet-extra#74]
which is the flipside of hopperMinecart8gtCooldown. Turns off the cooldown of hoppers.

**Comment by gnembon** [comment]
hmm, might get laggy on a bigger scale, don't you think? The delays are for something. I would suggest a configurable cooldown instead - more flexible.

**Comment by Neubulae** [comment]
> hmm, might get laggy on a bigger scale, don't you think? The delays are for something. I would suggest a configurable cooldown instead - more flexible.
 
that would be an absolutely greater idea!

**Comment by bencrazyalchemist** [comment]
Perhaps that would be better off as a customizable option of 2no's hopper optomization. Or if not it would be nice if it could be compatible.

**Comment by DragonEggBedrockBreaking** [comment]
> 
> 
> Perhaps that would be better off as a customizable option of 2no's hopper optomization.

Not really, as 2no2name's mod focuses on vanilla-behaviour optimisation, not adding mechanics to hoppers

**Comment by God-ly** [comment]
Additionally, it's not really as overpowered as you might think; one hopper can put in 72k items per hour, and because the items won't spread easily among the hoppers, its not an easy multiplication when you have more. Large scale, highly efficient farms still require a sorting system (ex. guardian farms, bartering systems, and pigmen farms).
Also, that's assuming you have enough storage for the hopper, as one hopper would fill at least 41-42 chests per hour. Without a sorting system, you still can't collect all the items.
It won't cause that much lag, if any, as you are removing entities (items) faster. Also, you wouldn't have to add more hoppers/hopper minecarts which decreases lag. Super sucker systems work fine and they can transfer way more than 1 item per tick.
The biggest benefit of this suggestion is not having to make these super suckers for mid-efficiency farms because one hopper can only support 9k items/hour. So it's more of a QOL convenience feature than an enhancement to vanilla.
